postgres/tasks.py

```diff
@@ -3,6 +3,9 @@
 from invoke import task, Responder
 
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
@Ruben-Hartog volgens PEP8 zijn alle variabelen met hoofdletters constanten (bestaat niet echt in Python, maar ze worden geacht nooit te veranderen). Meestal zet je deze constanten dan op module level, want ze zijn ook handig en constant voor functies in dezelfde file/module.

Reply:
Ah ja, klinkt logisch. Dit was ook een copy-paste van ergens anders 👼 , verder niet bij nagedacht.